tests/test_docker.py

```diff
@@ -0,0 +1,28 @@
+import subprocess
+import time
+from http import HTTPStatus
+
+import docker
+import requests
+
+SESSION_PORT = 9090
+
+
+def test_docker_container():
+    # Docker buildkit not supported by the Python SDK
+    command = ["docker", "build", "--tag", "template", "."]
+    subprocess.run(command, check=True)
```

Comment:
```suggestion
    subprocess.check_output(["docker", "build", "--tag", IMAGE_TAG, "."])
```

Where `IMAGE_TAG = "atoti-telemetry"` is defined next to `SESSION_PORT`.